client/state/data-layer/third-party/directly/index.js

```diff
@@ -3,21 +3,28 @@
  */
 import {
 	DIRECTLY_ASK_QUESTION,
-	DIRECTLY_INITIALIZE,
+	DIRECTLY_INITIALIZATION_START,
 } from 'state/action-types';
+import {
+	initializationCompleted,
+	initializationFailed,
+} from 'state/help/directly/actions';
 import * as directly from 'lib/directly';
 
 export function askQuestion( store, action, next ) {
 	directly.askQuestion( action.questionText, action.name, action.email );
 	next( action );
 }
 
-export function initialize( store, action, next ) {
-	directly.initialize();
+export function initialize( { dispatch }, action, next ) {
+	directly.initialize().then(
```

Comment:
This is probably a matter of preference but I prefer the `.then( ... ).catch( ... )` structure.